workflow-bot-app/src/test/java/com/symphony/bdk/workflow/IntegrationTestConfiguration.java

```diff
@@ -1,18 +1,106 @@
 package com.symphony.bdk.workflow;
 
 import com.symphony.bdk.workflow.engine.ResourceProvider;
+import com.symphony.bdk.workflow.engine.camunda.UtilityFunctionsMapper;
 
+import lombok.extern.slf4j.Slf4j;
+import org.camunda.bpm.engine.ProcessEngine;
+import org.camunda.bpm.engine.delegate.BpmnError;
+import org.camunda.bpm.engine.delegate.VariableScope;
+import org.camunda.bpm.engine.impl.cfg.ProcessEngineConfigurationImpl;
+import org.camunda.bpm.engine.impl.cfg.ProcessEnginePlugin;
+import org.camunda.bpm.engine.impl.el.ExpressionManager;
+import org.camunda.bpm.engine.impl.scripting.ExecutableScript;
+import org.camunda.bpm.engine.impl.scripting.env.ScriptingEnvironment;
+import org.camunda.bpm.engine.impl.util.ReflectUtil;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Profile;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
 
 import java.nio.file.Paths;
+import javax.script.Bindings;
+import javax.script.ScriptEngine;
+import javax.sql.DataSource;
 
 @TestConfiguration
+@EnableJpaRepositories
+@PropertySource("application-test.yaml")
+@Slf4j
+@EnableTransactionManagement
 @Profile("test")
-public class IntegrationTestConfiguration {
+public class IntegrationTestConfiguration implements ProcessEnginePlugin {
   @Bean("workflowResourcesProvider")
   public ResourceProvider workflowResourcesProvider() {
     return new TestResourcesProvider(Paths.get("dummy").toString());
   }
+
+  @Bean(name = "camundaBpmDataSource")
+  @ConfigurationProperties(prefix = "spring.camundadatasource")
+  public DataSource camundaDataSource() {
+    return DataSourceBuilder.create().build();
+  }
+
+  @Override
+  public void preInit(ProcessEngineConfigurationImpl processEngineConfiguration) {
```

Comment:
do we have to duplicate the code in integration test config?

Reply:
Initially the dev config was used in tests, but I had a duplicate conflict for the datasource bean. So I used profiles that's why I had to duplicate the config here.